src/main/java/com/commercetools/sync/categories/helpers/CategorySyncStatistics.java

```diff
@@ -16,7 +16,7 @@
      * Map that represents categories with missing parents; the keys of the map are the keys of the missing parent
      * categories and the value of each is a list of the children category keys.
      */
-    private Map<String, List<String>> categoryKeysWithMissingParents = new HashMap<>();
+    private Map<String, List<String>> categoryKeysWithMissingParents = new ConcurrentHashMap<>();
```

Comment:
Suggestion: I think it's better when you write comment for Map like this:
```
/**
     * Map that represents categories with missing parents
     *
     * key: keys of the missing parent categories
     * value: list of the children category keys
     */
```
The idea is to make clear what is key and what is value and not hide it in a long sentence.

Reply:
Why do you actually need `ConcurrentHashMap` in this class? I don't see you put anything into this map.

Reply:
@lojzatran https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/categories/CategorySync.java#L91 here it is set in the `CategorySync` class, and elements are put into this map concurrently in the [`CategorySync`](https://github.com/commercetools/commercetools-sync-java/blob/master/src/main/java/com/commercetools/sync/categories/CategorySync.java) class.

Reply:
> The idea is to make clear what is key and what is value and not hide it in a long sentence.

Definitely agree:  e3f0609. Thanks for the suggestion!

Reply:
> here it is set in the CategorySync class, and elements are put into this map concurrently in the CategorySync class.

agree with @lojzatran, the implementation looks weird: the value is update in one class, but stored in another. Looks obfuscated and error prone.